<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIImageViewer</class>
 <widget class="QMainWindow" name="UIImageViewer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>528</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btnPrevImage">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/go-previous.png</normaloff>:/icons/resources/go-previous.png</iconset>
        </property>
        <property name="shortcut">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNextImage">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/go-next.png</normaloff>:/icons/resources/go-next.png</iconset>
        </property>
        <property name="shortcut">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFitImage">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/fit-image.png</normaloff>:/icons/resources/fit-image.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRotateCCW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Rotate counter clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/reload.png</normaloff>:/icons/resources/reload.png</iconset>
        </property>
        <property name="shortcut">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRotateCW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Rotate clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/rotateCW.png</normaloff>:/icons/resources/rotateCW.png</iconset>
        </property>
        <property name="shortcut">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lCurrentImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lImageInfo">
        <property name="text">
         <string>Image Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSlideshow">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/media-playback-start.png</normaloff>:/icons/resources/media-playback-start.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbSlideshowPause">
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenWithExternalViewer">
        <property name="toolTip">
         <string>Open with external viewer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/open.png</normaloff>:/icons/resources/open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnPrevImage</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>displayPrevImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNextImage</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>displayNextImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFitImage</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>fitImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenWithExternalViewer</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>openWithExternalViewer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRotateCCW</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>rotateCCW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRotateCW</sender>
   <signal>clicked()</signal>
   <receiver>UIImageViewer</receiver>
   <slot>rotateCW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSlideshow</sender>
   <signal>toggled(bool)</signal>
   <receiver>UIImageViewer</receiver>
   <slot>toggleSlideshow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbSlideshowPause</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIImageViewer</receiver>
   <slot>setSlideshowTimeout(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayNextImage()</slot>
  <slot>displayPrevImage()</slot>
  <slot>fitImage()</slot>
  <slot>openWithExternalViewer()</slot>
  <slot>rotateCW()</slot>
  <slot>rotateCCW()</slot>
  <slot>toggleSlideshow(bool)</slot>
  <slot>setSlideshowTimeout(int)</slot>
 </slots>
</ui>
